projects/igniteui-angular/src/lib/drop-down/drop-down-item.base.ts

```diff
@@ -304,6 +304,17 @@ export class IgxDropDownItemBaseDirective implements DoCheck {
     clicked(event): void {
     }
 
+    /**
+     * @hidden
+     * @internal
+     */
+    @HostListener('mousedown', ['$event'])
+    handleMousedown(event): void {
```

Comment:
The event could be typed as well.
Also, add an accessor modifier, as I think lacking one triggers a lint warning. I know we'll go over all of these eventually, but might as well cross one from the ever-growing list
```suggestion
    public handleMousedown(event: MouseEvent): void {
```